dspace-api/src/main/java/org/dspace/content/factory/ContentServiceFactory.java

```diff
@@ -70,6 +74,14 @@
 
     public abstract SiteService getSiteService();
 
+    public abstract RelationshipTypeService getRelationshipTypeService();
```

Comment:
please, javadoc is required for new methods (note here but also for the following)

Reply:
This has been included in https://github.com/DSpace/DSpace/pull/2411